integration

```diff
@@ -315,6 +315,7 @@
   "finity69x2/nws_alerts",
   "firstof9/ha-openei",
   "FL550/dwd_weather",
+  "FlexMeasures/flexmeasures-ha-integration",
```

Comment:
PR is not editable.

Reply:
Closed and moved to https://github.com/hacs/default/pull/2354 to make it editable.